build/ci/release.yml

```diff
@@ -274,39 +275,18 @@ tasks:
       - func: "generate msi"
         vars:
           TOOL_NAME: ${tool_name}
-          unstable: -unstable
+          unstable: ${unstable}
       - func: "uninstall go-msi"
       - command: s3.put
         params:
           aws_key: ${aws_key}
           aws_secret: ${aws_secret}
           local_files_include_filter:
             - src/github.com/mongodb/mongodb-atlas-cli/dist/*.msi
-            - src/github.com/mongodb/mongodb-atlas-cli/dist/*.nupkg
           remote_file: ${project}/dist/${revision}_${created_at}/
           bucket: mongodb-mongocli-build
           permissions: public-read
           content_type: ${content_type|application/octet-stream}
-      - command: s3.put
-        params:
-          aws_key: ${aws_key}
-          aws_secret: ${aws_secret}
-          local_files_include_filter:
-            - src/github.com/mongodb/mongodb-atlas-cli/dist/*.nuspec
-          remote_file: ${project}/dist/${revision}_${created_at}/
-          bucket: mongodb-mongocli-build
-          permissions: public-read
-          content_type: ${content_type|text/xml}
-      - command: s3.put
-        params:
-          aws_key: ${aws_key}
-          aws_secret: ${aws_secret}
-          local_files_include_filter:
-            - src/github.com/mongodb/mongodb-atlas-cli/dist/tools/*.ps1
-          remote_file: ${project}/dist/${revision}_${created_at}/
-          bucket: mongodb-mongocli-build
-          permissions: public-read
-          content_type: ${content_type|text/plain}
```

Comment:
Just to make sure I understand the changes: Do we push the msi to an internal s3 bucket and then retrieve it in the gorelease step?

Reply:
discussed offline.